db/versions/0090.yml

```diff
@@ -0,0 +1,78 @@
+version: 90
+description: prepare queue for upcoming migration and refactoring
+migrationScript: |-
+  begin
+    alter table azure_queue_messages
+      add column task_queue_id text null, -- expose cryptic names
+      add column priority int null, -- don't forget about the priority
+      add column payload jsonb null; -- instead of base64 encoded string use jsonb
```

Comment:
Only thing going around in my head here is total size the jsonb is using up. If you don't think it'll be an issue, then np from me.

Reply:
i think should be fine, shouldn't take much more space than base64 encoded json that is used currently.
plus those new fields are not used in fetching, so wouldn't affect existing querying 👍 

but later it would be much easier to extract columns during second phase - https://github.com/taskcluster/taskcluster/pull/6554/files 

As I'm still playing with that part, I might discover that jsonb is not necessary, then I'll drop it here. 
The most problematic part is "cryptic" queue names for taskQueueId

Reply:
alright, I think I'm going to drop this afterall, thanks for making me rethink this :) 

I can also do 
```
select 
queue_name, 
convert_from(decode(message_text, 'base64'), 'utf-8')::jsonb->>'taskId',
convert_from(decode(message_text, 'base64'), 'utf-8')::jsonb->>'taskGroupId',
convert_from(decode(message_text, 'base64'), 'utf-8')::jsonb->>'schedulerId'
from azure_queue_messages
limit 100;
```
so no need to keep intermediary column 👍